src/main/jbake/content/community/docs/developer-guide/alerts-v2.adoc

```diff
@@ -872,6 +872,253 @@ Url can be a list of valid link:https://hc.apache.org/httpcomponents-core-ga/htt
 | Used for X-Proxy-Remote-User HTTP header.
 |=======================
 
+== Prometheus Integration
+
+=== Prometheus Alerter
+
+The Prometheus Alerter will listen for triggers tagged with "prometheus" tag. the Prometheus evaluations and interacts with the Alerts system. +
+ +
+The Prometheus Alerter uses the following conventions for trigger tags and context:
+
+[cols="^2,10"]
+|=======================
+| *Required* |
+_trigger.tags["prometheus"] // the value is ignored_ +
+ +
+| *Optional* |
+_trigger.context["prometheus.frequency"] = "<seconds between queries to Prometheus, default = 120>"_ +
+ +
+Note that the same frequency will apply to all Prometheus external conditions on the trigger
+| *Optional* |
+_trigger.context["prometheus.url"] = "<url, default = global setting>"_
+| *Required* |
+An ExternalCondition must be defined to be processed by the Prometheus External Alerter:
+ +
+externalcondition.alerterId = "prometheus" +
+externalcondition.expression = <BooleanExpression> | <ALERTSExpression> +
+|=======================
+
+=== Prometheus Alerter Endpoints
+
+|===
+| GET | /hawkular/alerter/status | link:#++GET_P8S_STATUS++[Status endpoint for Prometheus Alerter Integrations]
+| POST | /hawkular/alerter/notifications | link:#++POST_P8S_NOTIFICATIONS++[Listen Prometheus Notifications.]
```

Comment:
[Consume Prometheus Alertmanager WebHook Notifications]

Reply:
Ok, we can fix it here, but the source code was not documented, so I needed to guess a suitable name.